lib/tasks/populate.rake

```diff
@@ -96,9 +121,9 @@ namespace :pvb do
 
     desc 'populate visits for review apps or dev environments'
     task visits: :environment do
-      %w[Sudbury Pentonville Usk].each do |prison_name|
+      %w[Sudbury Pentonville Usk Aylesbury].each do |prison_name|
         prison = Prison.find_by!(name: prison_name)
-        visits = 0.upto(14).map { |_i|
+        visits = 0.upto(140).map { |_i|
```

Comment:
It's a good way to catch performance problem, i.e on the inbox page there are  two n+1 queries. I was able to catch it because i loaded a load of records. But i'm happy to revert it back to 14 or use an ENV var. 
We don't really care about the number of records on review apps, do we?